spec/models/generic_file_spec.rb

```diff
@@ -8,6 +8,16 @@
     @file.apply_depositor_metadata(user.user_key)
   end
 
+  it 'is initialized with blanks' do
+    subject.attributes.each do |k,v|
+      if v.is_a?(Array)
```

Comment:
Sounds good, I fixed per https://github.com/projecthydra/sufia/pull/948#discussion_r25710327 and squashed. Thanks.